### ISSUE TITLE
political science has repeat though in the field of science facet

### ISSUE THREAD

**Issue by kjgarza** [datacite/datacite#1818]
### Describe the bug

political_science appears twice in the array of results

```
{
  works(resourceTypeId:"Dataset" facetCount:100) {
    totalCount
    fieldsOfScienceCombined {
      id
      count
    }
  }
}

```

#### Expected Behaviour
political science would appear only once

#### Current Behaviour

<img width="1068" alt="Screenshot 2023-05-05 at 15 54 39" src="https://user-images.githubusercontent.com/1092861/236477786-19d3f47a-c95c-4de0-9a3c-ab4951f96030.png">

**Comment by jrhoads** [comment]
It looks like an issue with the underlying data.
Some of them are "Political Science" and some of them are "Political science"

If you include the title in the query, you can see this.  
```
{
  works(resourceTypeId:"Dataset" facetCount:100) {
    totalCount
    fieldsOfScienceCombined {
      id
      count
      title
    }
  }
}
```

I'm sure there are other places where this could happen.

Do we want to normalize the spelling in this field somehow?  This was not part of the spec but could be useful.

**Comment by antonninkov** [comment]
from a user perspective, if there were only 42 categories (1 for each OECD subdiscipline) that would be ideal. But I know there are also datasets categorized only under the high-level discipline as well, so ideally there would be only 48 categories with that limitation.

**Comment by xaucerr** [comment]
Hello @jrhoads,
While the bug is being resolved, I was wondering if there might be any workarounds or best practices you could suggest for users like myself. I agree with @antonninkov's comment about the ideal number of categories, but currently, the duplicate entries are causing some confusion in data interpretation​.
Any suggestions or temporary solutions (e.g. should we ignore one of them?) would be greatly appreciated as we navigate this issue. Thank you for your time and assistance.

**Comment by KellyStathis** [comment]
Hi @xaucerr - I think the duplicate "political_science" entries could be combined, as one represents "Political science" and one represents "Political Science", if you're seeing duplicate results like in @kjgarza's example.

Are you trying to query for a specific field of science? If you or @antonninkov have example queries, we'd be happy to review.

**Comment by antonninkov** [comment]
If you request the counts of datasets that have the fieldsofscienceCombined label, you get this:

`{
  "data": {
    "datasets": {
      "subjects": [
        {
          "title": "Chemical sciences",
          "count": 1266709
        },
        {
          "title": "Physical sciences",
          "count": 1194129
        },
        {
          "title": "Biological sciences",
          "count": 570300
        },
        {
          "title": "Earth and related environmental sciences",
          "count": 551941
        },
        {
          "title": "Computer and information sciences",
          "count": 244885
        },
        {
          "title": "Sociology",
          "count": 111562
        },
        {
          "title": "Health sciences",
          "count": 110711
        },
        {
          "title": "Clinical medicine",
          "count": 97858
        },
        {
          "title": "Economics and business",
          "count": 77937
        },
        {
          "title": "Psychology",
          "count": 74805
        },
        {
          "title": "Social sciences",
          "count": 68876
        },
        {
          "title": "Other social sciences",
          "count": 61179
        },
        {
          "title": "Educational sciences",
          "count": 54848
        },
        {
          "title": "Law",
          "count": 53048
        },
        {
          "title": "Political Science",
          "count": 51169
        },
        {
          "title": "Mathematics",
          "count": 41216
        },
        {
          "title": "Civil engineering",
          "count": 22123
        },
        {
          "title": "Electrical engineering, electronic engineering, information engineering",
          "count": 19006
        },
        {
          "title": "Materials engineering",
          "count": 17572
        },
        {
          "title": "Social and economic geography",
          "count": 17236
        },
        {
          "title": "Political science",
          "count": 16335
        },
        {
          "title": "Environmental engineering",
          "count": 15328
        },
        {
          "title": "Agricultural biotechnology",
          "count": 14779
        },
        {
          "title": "Other engineering and technologies",
          "count": 13161
        },
        {
          "title": "Media and communications",
          "count": 10886
        },
        {
          "title": "Languages and literature",
          "count": 9514
        },
        {
          "title": "Mechanical engineering",
          "count": 8582
        },
        {
          "title": "Agriculture, forestry and fisheries",
          "count": 6938
        },
        {
          "title": "Basic medicine",
          "count": 5546
        },
        {
          "title": "Medical engineering",
          "count": 4827
        },
        {
          "title": "History and archaeology",
          "count": 4438
        },
        {
          "title": "Chemical engineering",
          "count": 4396
        },
        {
          "title": "Philosophy, ethics and religion",
          "count": 3672
        },
        {
          "title": "Other agricultural sciences",
          "count": 3260
        },
        {
          "title": "Arts (arts, history of arts, performing arts, music)",
          "count": 2657
        },
        {
          "title": "Animal and dairy science",
          "count": 2440
        },
        {
          "title": "Other medical sciences",
          "count": 2029
        },
        {
          "title": "Veterinary sciences",
          "count": 1935
        },
        {
          "title": "Medical biotechnology",
          "count": 1641
        },
        {
          "title": "Humanities",
          "count": 1585
        },
        {
          "title": "Natural sciences",
          "count": 1549
        },
        {
          "title": "Agriculture, forestry, and fisheries",
          "count": 1526
        },
        {
          "title": "Nanotechnology",
          "count": 1268
        },
        {
          "title": "Other humanities",
          "count": 476
        },
        {
          "title": "Medical and health sciences",
          "count": 444
        },
        {
          "title": "Agricultural sciences",
          "count": 346
        },
        {
          "title": "Environmental biotechnology",
          "count": 241
        },
        {
          "title": "Other natural sciences",
          "count": 134
        },
        {
          "title": "Industrial biotechnology",
          "count": 130
        },
        {
          "title": "Engineering and technology",
          "count": 101
        },
        {
          "title": "Health biotechnology",
          "count": 30
        },
        {
          "title": "Nano-technology",
          "count": 3
        },
        {
          "title": "Chemical Science",
          "count": 1
        },
        {
          "title": "Glace souterraine",
          "count": 1
        },
        {
          "title": "Veterinary science",
          "count": 1
        }
      ]
    }
  }
}`

I notice a few issues including: 

Political Science and Political Sciences
Chemical Science and Chemical Sciences
Nanotechnology and nano-technology
Glace souterraine(?)

As well, I don't think this is something you can fix, but some are just labelled at the higher level OECD field (e.g., Natural Sciences), instead of the sub-field. Ideally they'd all have a sub-field associated.
